src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -225,6 +230,16 @@ function MoneyRequestView({report, betas, parentReport, policyCategories, should
                     />
                 </OfflineWithFeedback>
             )}
+            {shouldShowBillable && (
+                <View style={[styles.flexRow, styles.mb4, styles.justifyContentBetween, styles.alignItemsCenter, styles.ml5, styles.mr8]}>
+                    <Text color={!transactionBillable ? themeColors.textSupporting : undefined}>{translate('common.billable')}</Text>
```

Comment:
The conditional color of the label here led to this issue:
- https://github.com/Expensify/App/issues/53460

which was fixed by removing the style completely.